cfme/tests/openstack/cloud/test_volume_backup.py

```diff
@@ -0,0 +1,74 @@
+"""Tests for Openstack cloud volume Backups"""
+
+import fauxfactory
+import pytest
+
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.utils.log import logger
+
+pytestmark = [
+    pytest.mark.usefixtures("setup_provider_modscope"),
+    pytest.mark.provider([OpenStackProvider], scope='module')
```

Comment:
Required: provider should be function scope, remove scope='module'

Reply:
Will remove it and resubmit. just can you provide an explanation on the difference because it  worked like that.

Regarding PRT results, according to @jawatts, looks like it is a problem in PRT system

Reply:
BTW< i got below errors when remove scope:
peMismatch: You tried to access the 'function' scoped fixture 'provider' with a 'module' scoped request object, involved factories
cfme/fixtures/provider.py:281:  def setup_provider_modscope(request, provider)
../cfme_venv/lib/python2.7/site-packages/_pytest/fixtures.py:242:  def get_direct_param_fixture_func(request)
______________________ ERROR at setup of test_create_volume_incremental_backup[openstack-11] ______________________
ScopeMismatch: You tried to access the 'function' scoped fixture 'provider' with a 'module' scoped request object, invo

Reply:
I think you will have to use 
pytest.mark.usefixtures("setup_provider"),
pytest.mark.provider([OpenStackProvider]) 
both of them have to be function scoped I think .

Provider markers have to function scoped now because it takes care of adding/removing providers when necessary, and controlling number of providers per appliance .

Reply:
@sshveta is correct here, provider should always be marked function scoped. As for PRT, 5.9 should be working but 510 is still having issues.